### ISSUE TITLE
Release 4.0.0 causes crash

### ISSUE THREAD

**Issue by hjrb** [apexcharts/Blazor-ApexCharts#542]
ManagedError: Could not resolve field token 0x04000185, due to: Could not load type of field 'Program+<>c:<>9__0_3' (1) due to: Could not resolve type with token 01000020 from typeref (expected class 'ApexCharts.ApexChartsServiceOptions' in assembly 'Blazor-ApexCharts, Version=4.0.0.0, Culture=neutral, PublicKeyToken=null') assembly:Blazor-ApexCharts, Version=4.0.0.0, Culture=neutral, PublicKeyToken=null type:ApexCharts.ApexChartsServiceOptions member:(null) assembly:VSSApp.Client.dll type:<>c member:(null)
    at Jn (marshal-to-js.ts:349:18)
    at kr (invoke-cs.ts:277:19)
    at l.javaScriptExports.call_entry_point (managed-exports.ts:60:13)
    at Object.Fc [as runMain] (run.ts:44:45)
    at Object.callEntryPoint (blazor.webassembly.js:1:45490)
    at mn (blazor.webassembly.js:1:57708)
callEntryPoint	@	blazor.webassembly.js:1

**Comment by joadan** [comment]
Hi, is it possible for you to provide a minimal repo where this can be reproduced?

Thank you,

**Comment by hjrb** [comment]
Sure. Install the new version of the nuget package. Add the lines
@
```c#
builder.Services.AddApexCharts(e =>
            {
                e.GlobalOptions = new ApexChartBaseOptions
                {
                    Debug = true,
                    Theme = new Theme { Palette = PaletteType.Palette6 }
                };
            });
``` 
Run the application. The error occurs directly after the start. 
I have a Blazor client app created with .NET 7 so is not yet using the new render modes.

**Comment by joadan** [comment]
I just created a new 8.0 wasm project and was not able to reproduce the error.
Do you get the error if you try a new project?

**Comment by joadan** [comment]
@francis-j , Do you get the error if you create in a new  project?

**Comment by hjrb** [comment]
When I do a new App using the current templates it works just fine. I don't know if I can still create a new app with the old template.

**Comment by joadan** [comment]
Just to be sure, you have tried to clean your solution after updating to v4.0?

**Comment by hjrb** [comment]
OK. I cleaned the solution and then the error does not occur at startup right away. But when I open the page the uses ApexCharts the was a new error. This (new) error was about some routine missing in JS. So I forced a complete page refresh (CTRL+F5) and voilà  - it works. MS should really fix that. The only way to make sure a build really works is to completely delete all build artifacts forcefully. Rebuild all does NOT work B.A.D. But I am happy now. Thanks for the support and the great library.
